raiden/network/pathfinding.py

```diff
@@ -161,7 +161,7 @@ def get_pfs_info(url: str) -> PFSInfo:
             message=infos["message"],
             operator=infos["operator"],
             version=infos["version"],
-            confirmed_block_number=infos["network_info"]["confirmed_block"],
+            confirmed_block_number=infos["network_info"]["confirmed_block"]["number"],
```

Comment:
I think you need to adjust these tests as well: https://github.com/raiden-network/raiden/commit/291fde8733dc6b69d3dde7b9d1a035308263d88e